main.py

```diff
@@ -1,14 +1,18 @@
-import requests
-bestMovies =[]
-for i in range(1,26):
+from requests import Session
+
+sess = Session()
+
+best_movies = []

```

Comment:
Stick to the (python naming convention)[https://visualgit.readthedocs.io/en/latest/pages/naming_convention.html]